Update active-directory-conditional-access-technical-reference.md

benpahl opened the pull request:
added registry value to auto-deploy chrome extension needed for 1703 and above. BP

azurecla commented:
Hi __@rgardler__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rogardle). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@rgardler - I'll merge, but for future updates please use the private repo. You can easily get to the private repo by adding -pr to any public repo link:

  Public: https://github.com/Azure/azure-content/blob/master/articles/storsimple/storsimple-ova-deploy1-portal-prep.md

  Private: https://github.com/Azure/azure-content-pr/blob/master/articles/storsimple/storsimple-ova-deploy1-portal-prep.md

benpahl committed:
Update active-directory-conditional-access-technical-reference.md

added registry value to auto-deploy chrome extension needed for 1703 and above. BP
